### DISCUSSION TITLE
Email Templates

### DISCUSSION THREAD

**Discussion by crazyITnerd** [dani-garcia/vaultwarden#3628] [General]
Hi all, 
I have just recently setup a Vaultwarden server in Docker. 
When I get emails from the server, they go into my spam folder. Does anyone know why this is?
I am also wanting to create a few different email templates with my logo on it for when I send invites out. What are the steps to do this?

**Comment by stefan0xC** [comment]
> When I get emails from the server, they go into my spam folder. Does anyone know why this is?

That could depend on a number of things. e.g. #3572 but it could also depend on the anti-spam settings of your email provider (some being more protective than others, e.g. not liking if the `SMTP_FROM_NAME` differs too much from the local part) or the settings/reputation of your SMTP server (i.e. is it on a blacklist or do you have proper PTR records, SPF settings, DMARC setup, etc). You could check your mail server with sites like [mxtoolbox](https://mxtoolbox.com/).
edit: you might also want to check the headers of the messages in the spam folder (i.e. if there is a `X-Spam-Status` header field it might give more details).

>  I am also wanting to create a few different email templates with my logo on it for when I send invites out. What are the steps to do this?

If you just want to change the logos you could set `SMTP_EMBED_IMAGES=false` and overwrite the path for `/vw_static/logo-gray.png` with your reverse proxy. If you want [to change more in the templates](https://github.com/dani-garcia/vaultwarden/wiki/Translating-the-email-templates#how-to-translate--customize-the-templates), you could also overwrite the `email/email_header.hbs` file and change the URL in the template and other files. (But if you want the [embedded logo](https://github.com/dani-garcia/vaultwarden/blob/main/src/static/images/logo-gray.png) to change you'll have to recompile vaultwarden yourself.)
